test/e2e/exec_test.go

```diff
@@ -618,4 +618,15 @@ RUN useradd -u 1000 auser`, fedoraMinimal)
 		Expect(session).Should(ExitCleanly())
 		Expect(session.OutputToString()).To(Equal("root"))
 	})
+
+	It("podman exec with --no-session flag", func() {
+		session := podmanTest.RunTopContainer("no_session_test")
```

Comment:
You need a `SkipIfRemote()` on this test so it doesn't run with the remote client